api_proto.go

```diff
@@ -83,6 +85,28 @@ func (m *FileMatch) ToProto() *proto.FileMatch {
 	}
 }
 
+func (*FileMatch) Generate(rng *rand.Rand, _ int) reflect.Value {
```

Comment:
Do these generate methods usually live on the proto file? Maybe they should be in a test utils file instead?

Reply:
There is already another generator method there,  that's why I put them there.